examples/AlwaysSucceeds.purs

```diff
@@ -25,37 +27,107 @@ import Contract.Transaction
   runContract_ cfg $ do
     logInfo' "Running Examples.AlwaysSucceeds"
-    validator <- liftContractM "Invalid script JSON" $ alwaysSucceedsScript
+    validator <- liftContractM "Invalid script JSON" alwaysSucceedsScript
     vhash <- liftContractM "Couldn't hash validator" $ validatorHash validator
-    let
-      -- Note that CTL does not have explicit equivalents of Plutus'
-      -- `mustPayToTheScript` or `mustPayToOtherScript`, as we have no notion
-      -- of a "current" script. Thus, we have the single constraint
-      -- `mustPayToScript`, and all scripts must be explicitly provided to build
-      -- the transaction (see the value for `lookups` below as well)
-      constraints :: Constraints.TxConstraints Unit Unit
-      constraints = Constraints.mustPayToScript vhash unitDatum
-        $ Value.lovelaceValueOf
-        $ BigInt.fromInt 2_000_000
-
-      lookups :: Lookups.ScriptLookups PlutusData
-      lookups = Lookups.validator validator
-
-    ubTx <- liftedE $ Lookups.mkUnbalancedTx lookups constraints
-    BalancedSignedTransaction bsTx <-
-      liftedM "Failed to balance/sign tx" $ balanceAndSignTx ubTx
-    txId <- submit bsTx.signedTxCbor
-    logInfo' $ "Tx ID: " <> show txId
+    logInfo' "Attempt to lock value "
+    txId <- payToAlwaysSucceeds vhash validator
+    countToZero 20
+    logInfo' "Try to spend locked values"
+    spendFromAlwaysSucceeds vhash validator txId
+
+countToZero :: Int -> Contract () Unit
+countToZero n =
+  if n <= 0 then
+    pure unit
+  else do
+    logInfo' $ "Waiting before we try to unlock : " <> show n
+    liftAff <<< delay $ Milliseconds 1000.0
+    countToZero (n - 1)
+
+payToAlwaysSucceeds
+  :: ValidatorHash
+  -> Validator
+  -> Contract () TransactionHash
+payToAlwaysSucceeds vhash validator = do
+  let
+    constraints :: Constraints.TxConstraints Unit Unit
+    constraints = Constraints.mustPayToScript vhash unitDatum
+      $ Value.lovelaceValueOf
+      $ BigInt.fromInt 2_000_000
+
+    lookups :: Lookups.ScriptLookups PlutusData
+    lookups = Lookups.validator validator
+
+  buildSingnAndSubmitTx lookups constraints
+
+spendFromAlwaysSucceeds
+  :: ValidatorHash
+  -> Validator
+  -> TransactionHash
+  -> Contract () Unit
+spendFromAlwaysSucceeds vhash validator txId = do
+  let
+    scriptAddress = scriptHashAddress vhash
+  UtxoM utxos <-
+    fromMaybe (UtxoM Map.empty) <$> utxosAt scriptAddress
+
+  let
+    filteredById =
+      Map.filterWithKey hasTransactionId utxos
+
+  case fst <$> (head <<< Map.toUnfoldable) filteredById of
+    Just txInput ->
+      let
+        lookups :: Lookups.ScriptLookups PlutusData
+        lookups = Lookups.validator validator
+          <> Lookups.unspentOutputs utxos
+
+        constraints :: TxConstraints Unit Unit
+        constraints =
+          Constraints.mustSpendScriptOutput txInput unitRedeemer
+      in
+        void
+          $ buildSingnAndSubmitTx lookups constraints
```

Comment:
Again, I would keep this to one line `void $ buildSingnAndSubmitTx lookups constraints`